tutorials/export/exporting_basics.rst

```diff
@@ -239,46 +241,48 @@ changed:
 Setting a main scene
 --------------------
 
-The main scene is the one that your game will start in. In *Project -> Project
-Settings -> Application -> Run*, set *Main Scene* to "Main.tscn" by clicking
-the folder icon and selecting it.
+The main scene is the one that your game will start in. In
+**Project -> Project Settings -> Application -> Run**, set **Main Scene**
+to ``Main.tscn`` by clicking the folder icon and selecting it.
 
```

Comment:
Add

"In your project this will be a scene you will want your game to start on it does not have to be call Main.tscn you can name it whatever name you want. "

We don't want users to think they must name it Main.tscn for some reason